ci/ray_ci/serve.tests.yml

```diff
@@ -1 +1,2 @@
-flaky_tests: []
+flaky_tests:
+  - //python/ray/serve/tests:test_standalone_3
```

Comment:
CC: @edoakes , @sihanwang41 , this test has became very flaky (https://flakey-tests.ray.io/?owner=serve) so I jail it to unblock myself

Reply:
Got it, we'll fix